pkg/inventory/account_test.go

```diff
@@ -50,3 +50,24 @@ func TestAccountInventory(t *testing.T) {
 	assert.Len(t, inv.Delete, 1)
 	assert.Equal(t, "to-delete", inv.Delete[0].Name)
 }
+
+func TestAccountInventoryWithSameNameInstances(t *testing.T) {
+	create := []v1.ServiceAccount{{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "to-create",
+			Namespace: "tenant1",
+		},
+	}, {
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "to-create",
+			Namespace: "tenant2",
+		},
+	}}
+
+	inv := ForAccounts([]v1.ServiceAccount{}, create)
+	assert.Len(t, inv.Create, 2)
+	assert.Contains(t, create, create[0])
```

Comment:
Should this be testing `inv.Create [0]`?

Reply:
Good catch. It should be `assert.Contains(t, inv.Create, create[0])`